src/main/crossover.js

```diff
@@ -181,6 +181,7 @@ const lockWindow = ( lock, targetWindow = windows.win ) => {
 	const hideOnMouse = Number.parseInt( preferences.value( 'actions.hideOnMouse' ), 10 )
 	const hideOnKey = preferences.value( 'actions.hideOnKey' )
 	const tilt = checkboxTrue( preferences.value( 'actions.tiltEnable' ), 'tiltEnable' )
+	const resizeOnADS = Number.parseInt( preferences.value( 'actions.resizeOnADS' ), 0 )
```

Comment:
The second argument to Number.parseInt should be a radix value, I think it should be `10` rather than `0` (to set as base 10)

Reply:
My bad, must have been a typo